Runtime/Model/BacktraceConfiguration.cs

```diff
@@ -216,15 +217,37 @@ public class BacktraceConfiguration : ScriptableObject
         [Tooltip("If the database is unable to send its record, this setting specifies the maximum number of retries before the system gives up")]
         public int RetryLimit = 3;
 
+        [Tooltip("If the database is unable to send its record, this setting specifies the maximum number of retries before the system gives up")]
```

Comment:
I think you copy / pasted the tool tip from the above.  Make sure you update it to reflect the real use of the AttachmentPaths field @konraddysput

Reply:
ah good catch. Thanks